controllers/api/v1/token.go

```diff
@@ -0,0 +1,48 @@
+package apiv1
+
+import (
+	"errors"
+	"net/http"
+	"net/http/httptest"
+
+	"google-scraper/serializers"
+	. "google-scraper/services/oauth"
+
+	"github.com/tidwall/gjson"
+)
+
+type Token struct {
+	baseAPIController
+}
+
+func (c *Token) Create() {
+	writer := httptest.NewRecorder()
```

Comment:
I had to create this response writer to get the response, serialize it and rendering with `serveJSON `(found it in [so](https://stackoverflow.com/a/27988044)).  Because Oauth2's  `HandleTokenRequest` handles everything, it does not return a response, it renders the response itself which is non JSON API structure.

Reply:
I do like this approach 🙏